app/controllers/answers_controller.rb

```diff
@@ -37,6 +37,11 @@ def destroy
     end
   end
 
+  def accept
+    Answer.accept(params[:answer_id])
```

Comment:
Alternatively you could leverage the `before_action :set_answer` and then convert `:accept` to an _answer_ instance method